### DISCUSSION TITLE
Difference between API and script

### DISCUSSION THREAD

**Discussion by Remi1945** [Stellarium/stellarium#2409] [Q&A]
Hello
I have a script to check which star are in the field of view of a telescope. In this script I set the date with the function core.setJDay (Julian time) and the position with the core.setObserverLocation then I use the core.getObjectInfo to have the altitude and the elevation of a star.
If I do the same with the API I have differences for example the script gives 27.14/336.6 and the API 27.61/335.85. The difference is not big but they should be no difference

**Comment by gzotti** [comment]
API means Application Programming Interface. In this respect the Script language is API. What do you compare? Script API vs. HTTP API?

**Reply by Remi1945** [reply]
Yes I compare the result from a script API and a HTTP API

**Reply by Remi1945** [reply]
I have founded the problem.
In the HTTP Post the time value was badly formatted so I have a significative time difference between the given time in the script API and the given time in the HTTP API
